vector/src/main/res/values/strings.xml

```diff
@@ -2167,6 +2167,14 @@
     <string name="send_suggestion_sent">Thanks, the suggestion has been successfully sent</string>
     <string name="send_suggestion_failed">The suggestion failed to be sent (%s)</string>
 
+    <string name="send_feedback_space_title">Spaces feedback</string>
+    <string name="feedback">Feedback</string>
+    <string name="send_feedback_space_info">You’re using a beta version of spaces.  Your feedback will help inform the next versions.  Your platform and username will be noted to help us use your feedback as much as we can. To leave the beta, visit your settings.</string>
```

Comment:
There are extra spaces char: `spaces.  Your feedback` and `sions.  Your`. It happens when copy pasting from Figma.

Reply:
done